build.sbt

```diff
@@ -121,8 +122,25 @@ def formattingPreferences = {
     .setPreference(SpacesAroundMultiImports, true)
 }
 
+val defaultMultiJvmOptions: List[String] = {
+  import scala.collection.JavaConverters._
+  // multinode.D= and multinode.X= makes it possible to pass arbitrary
+  // -D or -X arguments to the forked jvm, e.g. -Dmultinode.Xmx512m
```

Comment:
> e.g. -Dmultinode.Xmx512m

Shouldn't that be `multinode.D=Xmx512m`? Sorry if I'm missing how this should be used